tools/diann/diann.xml

```diff
@@ -0,0 +1,486 @@
+<tool id="diann" name="DIA-NN" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>is a software for DIA/SWATH data processing</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.8.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <container type="docker">biocontainers/diann:v@TOOL_VERSION@_cv1</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir ./input_data &&
+        mkdir ./tmp &&
+
+        ## generate string like "--f file1 --f file2 ..." for each input in $input.f, which are comma separated
+        #set infiles_str = ''
+        #for $infile in $input.f
+            ## if brukertdf.d.tar, extract to ./input_data and append "./input_data/" to infiles_str
+            #if $infile.is_of_type("brukertdf.d.tar"):
```

Comment:
Should we add a bruker test, to be sure this actually works?

Reply:
Bruker files are quite large, even if one measures a blank in a very short gradient.. I'll try to generate one manually

Reply:
Yeah, I know. 2MB would be maybe also ok. But if we do not test this code path, it might not work :(

Reply:
I found one test file in the msconvert wrapper, but it's not functional with DIA-NN (raises a C++ error). I even tried to fix it... I do not know how to get a smaller file. Even if we measure a blank with a shorter gradient we get files ~500MB.

I will try to run the small file here as well, but I guess it will fail as well. I can confirm it's running with one of those files measured on our machine (timsTOF flex)

Reply:
![grafik](https://github.com/galaxyproteomics/tools-galaxyp/assets/34959927/66450437-31f0-4ef0-8a19-dec577df2c50)
the variable is not being expanded correctly when wrapped in quotes